### PR TITLE
Sanity tests: copy ignore-2.13.txt to ignore-2.14.txt

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY
https://github.com/ansible-collections/news-for-maintainers/issues/13

Dear maintainers,

I'd like to ask you to track the news-for-maintainers themselves in the future if possible

Thanks!

### CODE REVIEW

File: tests/sanity/ignore-2.14.txt [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+plugins/modules/azure_rm_containerregistryreplication_info.py validate-modules:undocumented-parameter
+plugins/modules/azure_rm_containerregistrywebhook.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_containerregistrywebhook.py validate-modules:doc-choices-do-not-match-spec
+plugins/modules/azure_rm_containerregistrywebhook.py validate-modules:doc-default-does-not-match-spec
+plugins/modules/azure_rm_containerregistrywebhook.py validate-modules:undocumented-parameter
+plugins/modules/azure_rm_containerregistrywebhook.py validate-modules:parameter-list-no-elements
+plugins/modules/azure_rm_containerregistrywebhook_info.py validate-modules:doc-default-does-not-match-spec
+plugins/modules/azure_rm_containerregistrywebhook_info.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_containerregistrywebhook_info.py validate-modules:undocumented-parameter
+plugins/modules/azure_rm_datalakestore.py validate-modules:doc-elements-mismatch
```
Comment (Fred-sun) [inline tests/sanity/ignore-2.14.txt]:
```suggestion

```

File: tests/sanity/ignore-2.14.txt [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+plugins/modules/azure_rm_datalakestore.py validate-modules:nonexistent-parameter-documented
+plugins/modules/azure_rm_deployment.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_deployment.py validate-modules:return-syntax-error
+plugins/modules/azure_rm_deployment_info.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_deployment_info.py validate-modules:return-syntax-error
+plugins/modules/azure_rm_dnsrecordset.py validate-modules:invalid-ansiblemodule-schema
+plugins/modules/azure_rm_dnszone.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_dnszone.py validate-modules:doc-elements-mismatch
```
Comment (Fred-sun) [inline tests/sanity/ignore-2.14.txt]:
```suggestion

```

File: tests/sanity/ignore-2.14.txt [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+plugins/modules/azure_rm_galleryimageversion.py validate-modules:doc-required-mismatch
+plugins/modules/azure_rm_galleryimageversion.py validate-modules:invalid-ansiblemodule-schema
+plugins/modules/azure_rm_galleryimageversion.py validate-modules:parameter-list-no-elements
+plugins/modules/azure_rm_image.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_image.py validate-modules:doc-required-mismatch
+plugins/modules/azure_rm_image.py validate-modules:parameter-list-no-elements
+plugins/modules/azure_rm_image_info.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_image_info.py validate-modules:parameter-list-no-elements
+plugins/modules/azure_rm_image_info.py validate-modules:return-syntax-error
+plugins/modules/azure_rm_iothub.py validate-modules:doc-elements-mismatch
```
Comment (Fred-sun) [inline tests/sanity/ignore-2.14.txt]:
```suggestion

```

File: tests/sanity/ignore-2.14.txt [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+plugins/modules/azure_rm_publicipaddress_info.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_publicipaddress_info.py validate-modules:parameter-list-no-elements
+plugins/modules/azure_rm_resource.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_resource.py validate-modules:parameter-list-no-elements
+plugins/modules/azure_rm_resource_info.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_resource_info.py validate-modules:parameter-list-no-elements
+plugins/modules/azure_rm_resourcegroup.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_resourcegroup_info.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_resourcegroup_info.py validate-modules:parameter-list-no-elements
+plugins/modules/azure_rm_registrationdefinition.py validate-modules:doc-elements-mismatch
```
Comment (Fred-sun) [inline tests/sanity/ignore-2.14.txt]:
```suggestion

```

File: tests/sanity/ignore-2.14.txt [hunk 5]
```diff
@@ -0,0 +1,331 @@
...
+plugins/modules/azure_rm_securitygroup.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_securitygroup.py validate-modules:undocumented-parameter
+plugins/modules/azure_rm_securitygroup.py validate-modules:doc-elements-mismatch
+plugins/modules/azure_rm_securitygroup.py validate-modules:doc-required-mismatch
+plugins/modules/azure_rm_securitygroup.py validate-modules:mutually_exclusive-unknown
+plugins/modules/azure_rm_securitygroup_info.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_securitygroup_info.py validate-modules:parameter-list-no-elements
+plugins/modules/azure_rm_storageaccount.py validate-modules:doc-missing-type
+plugins/modules/azure_rm_storageaccount.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_storageaccount.py validate-modules:doc-elements-mismatch
```
Comment (Fred-sun) [inline tests/sanity/ignore-2.14.txt]:
```suggestion

```

File: tests/sanity/ignore-2.14.txt [hunk 6]
```diff
@@ -0,0 +1,331 @@
...
+plugins/modules/azure_rm_storageaccount.py validate-modules:doc-required-mismatch
+plugins/modules/azure_rm_storageaccount_info.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_storageaccount_info.py validate-modules:parameter-list-no-elements
+plugins/modules/azure_rm_storageaccount_info.py validate-modules:return-syntax-error
+plugins/modules/azure_rm_storageblob.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_subnet.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_subnet.py validate-modules:parameter-list-no-elements
+plugins/modules/azure_rm_subnet_info.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_subscription_info.py validate-modules:doc-elements-mismatch
```
Comment (Fred-sun) [inline tests/sanity/ignore-2.14.txt]:
```suggestion

```

File: tests/sanity/ignore-2.14.txt [hunk 7]
```diff
@@ -0,0 +1,331 @@
...
+plugins/modules/azure_rm_loadbalancer.py validate-modules:doc-required-mismatch
+plugins/modules/azure_rm_loganalyticsworkspace.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_loganalyticsworkspace_info.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_loganalyticsworkspace_info.py validate-modules:parameter-list-no-elements
+plugins/modules/azure_rm_mariadbconfiguration_info.py validate-modules:return-syntax-error
+plugins/modules/azure_rm_mariadbdatabase.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_mariadbfirewallrule.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_mariadbserver.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_mariadbserver_info.py validate-modules:parameter-list-no-elements
+plugins/modules/azure_rm_monitorlogprofile.py validate-modules:doc-elements-mismatch
```
Comment (Fred-sun) [inline tests/sanity/ignore-2.14.txt]:
```suggestion

```

File: tests/sanity/ignore-2.14.txt [hunk 8]
```diff
@@ -0,0 +1,331 @@
...
+plugins/modules/azure_rm_sqlserver_info.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_trafficmanager.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_trafficmanager.py validate-modules:doc-choices-do-not-match-spec
+plugins/modules/azure_rm_trafficmanager.py validate-modules:doc-default-does-not-match-spec
+plugins/modules/azure_rm_trafficmanager.py validate-modules:invalid-documentation
+plugins/modules/azure_rm_trafficmanager.py validate-modules:undocumented-parameter
+plugins/modules/azure_rm_trafficmanager.py validate-modules:doc-elements-mismatch
+plugins/modules/azure_rm_trafficmanager.py validate-modules:doc-required-mismatch
+plugins/modules/azure_rm_trafficmanagerendpoint.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_trafficmanagerendpoint.py validate-modules:doc-elements-mismatch
```
Comment (Fred-sun) [inline tests/sanity/ignore-2.14.txt]:
```suggestion

```

File: tests/sanity/ignore-2.14.txt [hunk 9]
```diff
@@ -0,0 +1,331 @@
...
+plugins/modules/azure_rm_virtualnetworkpeering.py validate-modules:doc-required-mismatch
+plugins/modules/azure_rm_virtualnetworkpeering_info.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_virtualnetworkgateway.py validate-modules:parameter-type-not-in-doc
+plugins/modules/azure_rm_virtualnetworkgateway.py validate-modules:doc-choices-do-not-match-spec
+plugins/modules/azure_rm_virtualnetworkgateway.py validate-modules:doc-default-does-not-match-spec
+plugins/modules/azure_rm_virtualnetworkgateway.py validate-modules:doc-missing-type
+plugins/modules/azure_rm_virtualnetworkgateway.py validate-modules:doc-elements-mismatch
+plugins/modules/azure_rm_virtualnetworkgateway.py validate-modules:doc-required-mismatch
+plugins/modules/azure_rm_vpnsite.py validate-modules:invalid-ansiblemodule-schema
+plugins/modules/azure_rm_vpnsite.py validate-modules:doc-elements-mismatch
```
Comment (Fred-sun) [inline tests/sanity/ignore-2.14.txt]:
```suggestion

```

**Review by Fred-sun — COMMENTED** [review]
Ansible2.14@dev, The document detection has been changed, please make the following changes. Thank you very much!

**Comment by Andersson007** [comment]
> Ansible2.14@dev, The document detection has been changed, please make the following changes. Thank you very much!

@Fred-sun committed, thanks, PTAL

**Comment by xuzhang3** [comment]
LGTM, shipping it ⛵

**Pull request merged**